src/templates/dandelion/index.js

```diff
@@ -0,0 +1,228 @@
+        const {
+          domain,
+          voting,
+          tokens,
+          lock,
+          redemptions,
+          tokenRequest,
+        } = props.data
+        return (
+          <ReviewScreen
+            screenProps={props}
+            items={[
+              {
+                label: 'General info',
+                fields: [
+                  ['Organization template', 'Dandelion'],
+                  ['Name', completeDomain(domain)],
+                ],
+              },
+              {
+                label: (
+                  <KnownAppBadge
+                    appName="dandelion-voting.aragonpm.eth"
+                    label="Voting"
+                  />
+                ),
+                fields: VotingScreen.formatReviewFields(voting),
+              },
+              {
+                label: (
+                  <KnownAppBadge
+                    appName="token-manager.aragonpm.eth"
+                    label="Tokens"
+                  />
+                ),
+                fields: TokensScreen.formatReviewFields(tokens),
+              },
+              {
+                label: (
+                  <KnownAppBadge
+                    appName="time-lock.aragonpm.eth"
+                    label="Time lock"
+                  />
+                ),
+                fields: LockScreen.formatReviewFields(lock),
+              },
+              {
+                label: (
+                  <KnownAppBadge
+                    appName="redemptions.aragonpm.eth"
+                    label="Redemptions"
+                  />
+                ),
+                fields: RedemptionsScreen.formatReviewFields(redemptions),
+              },
+              {
+                label: (
+                  <KnownAppBadge
+                    appName="token-request.aragonpm.eth"
+                    label="Token Request"
+                  />
+                ),
+                fields: TokenRequestScreen.formatReviewFields(tokenRequest),
+              },
+            ]}
+          />
+        )
+      },
+    ],
+  ],
+  prepareTransactions(createTx, data) {
+    const blockTime = getBlockTime()
+    const {
+      domain,
+      optionalApps = [],
+      tokens,
+      voting,
+      lock,
+      redemptions,
+      tokenRequest,
+    } = data
+    const useAgentAsVault = optionalApps.includes('agent.aragonpm.eth')
+
+    // Tokens app
+    const { tokenName, tokenSymbol, members } = tokens
+    const baseStake = new BN(10).pow(new BN(18))
+    const stakes = members.map(([_, stake]) =>
+      baseStake.mul(new BN(stake.toString())).toString()
+    )
+    const accounts = members.map(([account]) => account)
+
+    // Finance app
+    const financePeriod = 0 // Fallback to default 30 days
+
+    // Voting app
+    const { support, quorum, duration, buffer, delay } = voting
+    const adjustedSupport = onePercent.mul(new BN(support)).toString()
+    const adjustedQuorum = onePercent.mul(new BN(quorum)).toString()
```

Comment:
It has the same behavior as the original Voting app, so  with those values would not fail the Org creation.

Reply:
In this regard it's the same as the original voting app so it would only fail with 100% support ?

Reply:
It would be for both, as quorum must be lower than support. We can add a check like in the other templates: https://github.com/aragon/aragon/blob/master/src/templates/company/index.js#L20